README.md

```diff
@@ -132,7 +132,10 @@ To set options and keep lint-staged extensible, advanced format can be used. Thi
 
 ## Filtering files
 
-It is possible to run linters for certain paths only by using [minimatch](https://github.com/isaacs/minimatch) patterns. The paths used for filtering via minimatch are relative to the directory that contains the `.git` directory. The paths passed to the linters are absolute to avoid confusion in case they're executed with a different working directory, as would be the case when using the `gitDir` option.
+It is possible to run linters for certain paths only by using glob patterns. [minimatch](https://github.com/isaacs/minimatch) is used to filter the staged files according to these patterns.
+File patterns should be specified _relative to the `package.json` location_ (i.e. where `lint-staged` is installed).
```

Comment:
LGTM, @okonet has approved this PR and I think we can now join these sentences back into one line

Reply:
after that, feel free to merge this PR. thank you @sudo-suhas 🙏

Reply:
I have intentionally kept these in separate sentences. I mentioned this earlier, not sure if you saw it. The rule of thumb I am following is, if any sentence exceeds 80 chars, start the next sentence with a new line. This way, git diffs are more helpful when we change anything in the para. And there is absolutely no difference in the way it is rendered. That said, I don't particularly mind keeping it in a single sentence if you prefer. What do you say?

Reply:
yeah, I did read your comment describing it before.

> The rule of thumb I am following is, if any sentence exceeds 80 chars, start the next sentence with a new line. This way, git diffs are more helpful when we change anything in the para. And there is absolutely no difference in the way it is rendered.

hmm, I doubt that, see https://github.com/airbnb/javascript/pull/1691/files for example. the diffs are clear and even more helpful

Reply:
I made it into a single line but I don't see the diff similar to the link you had shared. It is probably applicable for short sentences which are mostly similar after the change.